src/Components/EventListPage.js

```diff
@@ -57,7 +58,7 @@ class EventListPage extends Component {
     }
 
     handleChangeRegion(e) {
-        this.setState({ location: e.target.textContent }, this.setNewEvents);
+        this.setState({ location: e.target.children[1].innerText}, this.setNewEvents);
```

Comment:
I dont think we can just do this anymore. Because we are using location state as the query param to the backend. So if we are using canonical names in the backend for location and you send "San Francisco", the template is "SF" and it won't know to fetch that.